examples/model_selection/plot_nested_cross_validation_iris.py

```diff
@@ -0,0 +1,119 @@
+"""
+=========================================
+Nested versus Non-Nested cross-validation
+=========================================
+
+This example compares non-nested and nested cross-validation strategies on a
+classifier of the iris data set. Nested cross-validation (CV) is often used to
+train a model in which hyperparameters also need to be optimized. Nested CV
+approximates the generalization error of the resulting estimator of the
+hyperparameter search. It is generally best practice to use nested CV as
+non-nested CV will sometimes provide a slightly more biased and optimistic
```

Comment:
Rereading "them" (i.e. the parameters) would have done instead of it. I didn't write that with the intention that you would necessarily copy verbatim, so your edits to my paraphrases are welcome. Even when they arbitrarily side with Webster.